Mironov Alexey

m1ralx opened the pull request.

m1ralx committed:
Картинки двигаются

m1ralx committed:
+solution

m1ralx committed:
+./Photos

m1ralx committed:
translateX fixes
add small delay

m1ralx committed:
codestyle fixes

m1ralx committed:
oh wait..newline at end of file

VorontsovMaxim commented on index.css:
```diff
@@ -1 +1,139 @@
+    height: 180px;
+    overflow: hidden;
+    transition: all 1s cubic-bezier(.1, .01, .5, 1.2);
+    transition-delay: .3s;
```
`transition-delay` можно 4ым в `transition` писать

VorontsovMaxim commented on index.css:
```diff
@@ -1 +1,139 @@
+        -10px 0 20px rgba(0, 0, 0, .3);
+}
+
+.gallery article[class^='image'] img
```
Чтобы не писать таки длинные (и не очень быстрые) селекторы, можно всем картинкам дать класс

VorontsovMaxim commented on index.css:
```diff
@@ -1 +1,139 @@
+
+#button-1:checked ~ div
+{
+    transform: translateX(30%);
```
При выборе разных картинок - они встают в разные места по горизонтали. Да и размер экрана на это влияет.  Размер плитки с картинкой ведь у нас фиксирован. Может сделаем так, чтобы всегда вставали в одно место?

VorontsovMaxim commented:
Еще есть вот такая беда, которая воспроизводится когда выбирать другую картинку в то время как анимация еще не закончилась
![wow](https://cloud.githubusercontent.com/assets/14827792/11427750/324aa378-9487-11e5-8544-a2e582b22967.png)

VorontsovMaxim commented:
А в целом хорошо)

VorontsovMaxim commented:
Допы будут?

m1ralx commented:
@VorontsovMaxim класс для картинок добавлю. аргументы у transform'а исправлю. Наверное смогу придумать как спозиционировать, чтобы в одно место вставали картинки. Анимацию можно поправить наверное, добавив ей delay небольшой, но это пока предположение ( и, даже если сработает, то наверное костыль)

m1ralx commented:
@VorontsovMaxim допов не будет :(

VorontsovMaxim commented:
Насчет бага с анимацией. Delay - не решение) Поэксперементируй с перспективой. Подумай - действительно ли ее нужно каждый раз выставлять на `:checked`, а затем еще и анимировать (`transition: all ...`)?

m1ralx committed:
Теперь выбранные картинки встают на одно место.
Добавил класс для картинок -> упростил селекторы
Перспектива теперь установлена у класса галереи, кажется, что больше такого бага нет

m1ralx commented:
@VorontsovMaxim Вроде всё поправил.

m1ralx committed:
transition delay -> transition

m1ralx committed:
leading zero :\

m1ralx committed:
+justify-content: center
-margin
+min-width

VorontsovMaxim commented on index.html:
```diff
@@ -1,11 +1,57 @@
+        </article>
+        <article class="image-5">
+            <label for="button-5">
+                <img src="./Photos/ogurec.jpg">
```
Обделил огурца чего-то классом `gallery-item` и он теперь не влазит

m1ralx commented:
@VorontsovMaxim Убрал дискриминацию огурца

m1ralx committed:
fixed cucumber

VorontsovMaxim commented:
Принято
